river/Config.zig

```diff
@@ -51,6 +51,16 @@ pub const HideCursorWhenTypingMode = enum {
     enabled,
 };
 
+pub const ViewPosition = struct {
+    x: c_int,
+    y: c_int,
+};
+
+pub const ViewDimension = struct {
```

Comment:
Oh, and I'd just call these `Position` and `Dimensions`. There's nothing about them that is view-specific.